saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/MultiModalSpRLApp.scala

```diff
@@ -102,7 +102,7 @@ object MultiModalSpRLApp extends App with Logging {
         classifier.load()
         if (classifier == TrajectorPairClassifier || classifier == LandmarkPairClassifier) {
           if (!pairsPopulated) {
-            populatePairDataFromAnnotatedCorpus(xmlReader, isTrain, x => IndicatorRoleClassifier(x) == "Indicator")
+            populatePairDataFromAnnotatedCorpus(xmlReader, isTrain, x => SentenceLevelConstraintClassifiers.IndicatorConstraintClassifier(x) == "Indicator")
```

Comment:
if you remember when we used the train/test setting it wasn't better than unconstrained one, so we decided to use that.